client_test.go

```diff
@@ -228,3 +229,12 @@ func TestClientZeroLengthPacket(t *testing.T) {
 		c.Close()
 	}
 }
+
+func TestClientShortPacket(t *testing.T) {
+	// init packet too short.
+	packet := []byte{0, 0, 0, 1, 2}
+
+	r := bytes.NewReader(packet)
+	_, err := NewClientPipe(r, &sink{})
+	require.True(t, errors.Is(err, errShortPacket))
```

Comment:
The output of this `testify` “convenience function” is not actually at all useful:
```
--- FAIL: TestClientShortPacket (#.##s)
     client_test.go:239
                          Error Trace: client_test.go:239
                          Error:       Should be true
                          Test:        TestClientShortPacket
```